### ISSUE TITLE
Hyper-V is strictly required for everything

### ISSUE THREAD

**Issue by ygoe** [dotnet/docs-maui#413]
The page says: "Hardware acceleration must be enabled to maximize Android emulator performance."

This isn't accurate. Hyper-V must be installed and enabled or the Android emulator will not even run! The emulator never worked for me on a Surface Pro 4, it always failed with a short message that I don't understand (and that was shown for less than a second). Today I revisited the documentation and found this page. I checked and noticed that both shown Hyper-V components were not installed. After activating them and restarting Windows, the Android emulator would suddenly start and I can interact with the virtual device. Debugging an app also works now.

So this is clearly a strict requirement, not just something that might enhance performance. The documentation should reflect that to make clear what is necessary.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 779d2db3-3e9c-2692-57ff-787d9465a444
* Version Independent ID: e77ba6a7-e59d-1e1e-52e1-935f98a985cf
* Content: [Build your first .NET MAUI app - .NET MAUI](https://docs.microsoft.com/en-us/dotnet/maui/get-started/first-app?pivots=devices-android)
* Content Source: [docs/get-started/first-app.md](https://github.com/dotnet/docs-maui/blob/main/docs/get-started/first-app.md)
* Product: **dotnet-mobile**
* Technology: **dotnet-maui**
* GitHub Login: @davidbritch
* Microsoft Alias: **dabritch**

---
[Associated WorkItem - 90444](https://dev.azure.com/msft-skilling/Content/_workitems/edit/90444)

**Comment by adegeo** [comment]
Hello @ygoe did you enable HAXM when Hyper-V wasn't installed/running? I think the statements in the article are misleading as it seems the latest android emulator does require either HAXM or WHPX, at least according to Google's documentation https://developer.android.com/studio/run/emulator-acceleration

**Comment by davidbritch** [comment]
@ygoe 

It is possible to launch the Android emulator without having a hypervisor installed. I've done it myself. However, it took about 45 mins to launch the emulator and app!

There are actually three possible hypervisors for accelerating emulator performance, and depending on your chipset it maybe necessary to disable Hyper-V. So afraid it's incorrect to say that Hyper-V must be installed and enabled. It's more complicated than that.

However, agreed that the doc needs to be clearer.

**Comment by ygoe** [comment]
Well, okay, if there are ways to use it without Hyper-V, then I'd love to learn about them. This requirement is just what I found out. That happens when you let users help writing the documentation. ;-)

**Comment by ygoe** [comment]
I've revisited this after installing the MAUI bits on my main computer after upgrading to VS 17.4 (.NET 7). It looks like the documentation has been enhanced since then. There's a subtle [web link](https://learn.microsoft.com/en-us/dotnet/maui/get-started/first-app?view=net-maui-7.0&tabs=vswin&pivots=devices-android) below the VS toolbar after creating a MAUI project that also covers setting up the Android emulator. There are more links in that article.

I learned that HAXM is required when Hyper-V is not used, and that it's something from Intel that I have to install. I didn't like installing more Intel stuff. The last thing I installed was the latest GPU driver, now my system boots with a forced input method language toolbar that I have to fight against every time. But without that driver, the Signal Desktop app wouldn't start anymore. Maybe my 10 years old Core i7 needs a replacement? But it's still working totally fine!

I also learned that without HW acceleration I cannot use x64 Android images. But the last x86 Android is version 11, so this approach probably has no future.

I enabled Hyper-V again. When I need to use VMware or another such app, I'll have to disable it and reboot. The Hyper-V architecture is just annoying.
